bm-persona/Dining/DiningDataSource/DiningHalls/DiningHallDataSource.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  DiningHallDataSource.swift
+//  bm-persona
+//
+//  Created by Kevin Hu on 3/14/20.
+//  Copyright © 2020 RJ Pimentel. All rights reserved.
+//
+
+import Foundation
+import Firebase
+
+fileprivate let kDiningHallEndpoint = "Dining Halls"
+
+class DiningHallDataSource: DataSource {
+    
+    // Returns the collection name for the given date. Must match the name on Firebase.
+    private static func collectionForDate(_ date: Date) -> String {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "yyyy-MM-dd"
```

Comment:
We should standardize which dateFormat we use. I think I was using MM/dd/yyyy

Reply:
I did this format because I use it to get the collection for the day's menus from Firebase, which has this format. Unsure if there's a better way to query the proper collection.